text/trace/0235-sampling-threshold-in-trace-state.md

```diff
@@ -0,0 +1,87 @@
+# Sampling Threshold Propagation in TraceState
+
+## Motivation
```

Comment:
please add a discussion of calculating the weight for each sample

Reply:
I feel like I'd rather include some relatively generic code showing conversion between the different modes, rather than referring to a full implementation. In other words, I don't think we need executable code or full error checking, just a clear example of the math. Do you agree?

Reply:
yes, I just want to see math, not the code

Reply:
Please see the section on Algorithms below and let me know what you think.